wiki/Guides/How_to_Use_the_Offset_Wizard/de.md

```diff
@@ -0,0 +1,26 @@
+# Wie man den Offset-Assistenten benutzt
+
+*Siehe auch: [Offset-Assistent](/wiki/Client/Options/Offset_Wizard) und [Universales Offset](/wiki/Client/Options/Universal_offset)*
+
+Wenn die [Hitsounds](/wiki/Beatmapping/Hitsound) in jeder gespielten Beatmap nicht mit der Musik synchron sind, ist es sehr wahrscheinlich, dass du dein [universales Offset](/wiki/Client/Options/Universal_offset) umstellen musst. Am einfachsten geht das mit dem **Offset-Assistenten**.
+
+Führe die folgenden Schritte aus, um dein universales Offset mit Hilfe des Offset-Assistenten anzupassen.
+
+## Den Offset-Assistenten öffnen
+
+1. Öffne osu!.
+2. Gehe zum Menü `Optionen` oder drücke `Strg` + `O`.
+3. Suche nach `offset` in der Schnellsuche und klicke auf den Button `Offset-Assistent starten`.
```

Comment:
just a note. the English version uses double quotes here ("offset") but the inline code notation in the other article, so it's probably best to leave it.

Reply:
Yes, I was aware of that and wanted to keep it consistent with the other articles.